components/brave_rewards/resources/page/reducers/wallet_reducer.ts

```diff
@@ -222,6 +222,27 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
       }
 
       state.externalWallet = action.payload.wallet
+      break
+    }
+    case types.GET_MONTHLY_REPORT: {
+      chrome.send('brave_rewards.getMonthlyReport', [
+        action.payload.month,
+        action.payload.year
+      ])
+      break
+    }
+    case types.ON_MONTHLY_REPORT: {
+      state = { ...state }
```

Comment:
if you want to test it out you can print out `action.payload.report` to see if data is there. Sadly this is just a preparation for final PR, so there is no test plan as you can't test it in UI. Tests will be added when final PR is done.